RIOT-OS/RIOT
# .murdock: Fix kconfig hash check

### Contribution description

It seems like #18236 introduced some errors in the kconfig hash check.
Running locally I get a `.murdock: line 480: [: too many arguments` message.
On the CI it looks like `./.murdock: 480: [: 4994c097e49decc77a99c2254e46c710238dc4f2: unexpected operator`.
I noticed that the path names are now included in the comparison.
It seems surrounding them in quotes helps but this also compares whitespaces and newlines...



### Testing procedure

Check the murdock output in `tests/shell` for `native` and ensure no errors on line 480.


### Issues/PRs references
Introduced in #18236
